src/init.c

```diff
@@ -37,6 +39,24 @@ extern void _PG_fini(void);
 void
 _PG_init(void)
 {
+	if (!process_shared_preload_libraries_in_progress)
+	{
+	
+		char	   *force_load = GetConfigOptionByName("timescaledb.force_load", NULL, true);
+		if (force_load == NULL || strlen(force_load) != 2 || strncmp(force_load, "on", 2) != 0) {
+			char	   *config_file = GetConfigOptionByName("config_file", NULL, false);
+			elog(ERROR,
```

Comment:
Yeah I put a comment above as to whether the long text should be in the error message or the hint. The problem with hint, is it's not always displayed (according to client settings). But you think it should be in the hint?